### DISCUSSION TITLE
Show routes.php routes in Control Panel

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#9508] [Ideas]
> Created by: Leevi Graham (leevi@newism.com.au) on 2016/03/09 13:08:47 +0000
> Votes at time of UserVoice import: 1

-----------------------------------

It would be great if routes defined in routes.php were visible in the routes settings section of the CP.

**Comment by juddlyon** [comment]
Showing the routes in the CP would be ideal, but even a message that says routes.php is in use would suffice. Currently it's a bit of a blind spot, especially if you are working on a site you did not create.

**Comment by piotrpog** [comment]
There could also be list of Twig files that can be used as routes - without "_" at the start of filename. I recently encountered the project where there were control panel routes, template routes, file routes, it was all kinda mixed and difficult to keep track of.
